tola/util.py

```diff
@@ -118,6 +119,27 @@ def user_to_tola(backend, user, response, *args, **kwargs):
         userprofile.save()
 
 
+def auth_allowed(backend, details, response, *args, **kwargs):
```

Comment:
Wouldn't it better to have it inside a new `auth.py` or `auth_pipeline.py` file? This util file which has everything mixed I don't think it's a good idea.

Reply:
I agree! So, we should move the `def user_to_tola()` function as well and update the auth pipeline.